src/OrchardCore/OrchardCore.Users.Core/Services/IdentityOrchardHelperExtensions.cs

```diff
@@ -0,0 +1,83 @@
+using System;
+using System.Collections.Generic;
+using System.Security.Claims;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.Extensions.DependencyInjection;
+using OrchardCore.Security.Permissions;
+using OrchardCore.Users;
+
+namespace OrchardCore;
+
+public static class IdentityOrchardHelperExtensions
+{
+    public static async Task<IUser> UserAsync(this IOrchardHelper helper, UserManager<IUser> userManager = null)
+    {
+        userManager ??= helper.HttpContext.RequestServices.GetRequiredService<UserManager<IUser>>();
+
+        return await userManager.FindByIdAsync(await helper.UserIdAsync());
+    }
+
+    public static async Task<bool> AuthorizeAsync(this IOrchardHelper helper, Permission permission, IAuthorizationService authorizationService = null)
```

Comment:
I dislike adding `IAuthorizationService`, it could be an extension for `IAuthorizationService` instead, also we should check if we don't have this one already